testutil/sample/crosschain.go

```diff
@@ -54,6 +54,36 @@ func RateLimiterFlags() types.RateLimiterFlags {
 	}
 }
 
+func RateLimiterFlagsFromRand(r *rand.Rand) types.RateLimiterFlags {
```

Comment:
We can update the function above
```
func RateLimiterFlags() types.RateLimiterFlags {
	RateLimiterFlagsFromRand(Rand())
}
```
Might be the same for other function in the file

Reply:
Yeah we should infact do that for most functions in the sample directory , I would like to take that on as a spearate PR 
https://github.com/zeta-chain/node/issues/3266